frontend/src/components/professor/ChapterDetails/ChapterDetails.js

```diff
@@ -123,7 +149,11 @@ function ChapterDetails() {
                           ({activity.posX}, {activity.posY})
                         </td>
                         <td>Pkt: {activity.points}</td>
-                      </tr>
+                        <td>
+                          <FontAwesomeIcon icon={faPenToSquare} onClick={() => startActivityEdition(activity)} />
+                          <FontAwesomeIcon icon={faTrash} onClick={() => deleteActivity(activity)} />
+                        </td>
+                      </TableRow>
```

Comment:
Myślę że będzie fajniej jeśli damy to na oddzielne pola:
```suggestion
                        <td>
                          <FontAwesomeIcon icon={faPenToSquare} onClick={() => startActivityEdition(activity)} />
                        </td>
                        <td>
                          <FontAwesomeIcon icon={faTrash} onClick={() => deleteActivity(activity)} />
                        </td>
                      </TableRow>
```